src/main/java/com/zenjava/javafx/maven/plugin/WebMojo.java

```diff
@@ -198,6 +205,16 @@ public void execute() throws MojoExecutionException, MojoFailureException {
             String embeddedWidth = this.embeddedWidth != null ? this.embeddedWidth : String.valueOf(width);
             String embeddedHeight = this.embeddedHeight != null ? this.embeddedHeight : String.valueOf(height);
             deployParams.setEmbeddedDimensions(embeddedWidth, embeddedHeight);
+            
+            if (templates != null)
+                for (String template : templates) {
+                  final File in = new File(project.getBasedir(), "src/main/deploy/templates/" + template);

```

Comment:
Will be better to use OS specific path separators i.e. use [Path#get](http://docs.oracle.com/javase/7/docs/api/java/nio/file/Paths.html#get%28java.lang.String,%20java.lang.String...%29).